src/OpenRiaServices.Hosting.AspNetCore/Test/OpenRiaServices.Hosting.AspNetCore.Test/OpenRiaServices.Hosting.AspNetCore.Test.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="MSTest.TestFramework" Version="2.2.7" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\OpenRiaServices.Server\Test\OpenRiaServices.Server.Test.csproj" />
```

Comment:
there is always a risk of tests executing multiple times when referencing test projects from each other.

It might be better to link shared files or use a common shared library project such as / similar to "OpenRiaServices.Common" .projects ..

Reply:
I think you should be able to access most DomainServices by includingOpenRiaServices.Common.DomainServices.Test